bin/ruby-build

```diff
@@ -106,6 +106,15 @@ is_mac() {
   [ $# -eq 0 ] || [ "$(osx_version)" "$@" ]
 }
 
+is_freebsd() {
+  [ "$(uname -s)" = "FreeBSD" ]
+}
+
+freebsd_package_prefix() {
+  local package="$1"
+  echo "$(pkg info --prefix "$package" 2>/dev/null | cut -wf2)"
```

Comment:
How about just unwrapping this instead of capturing the output and echo-ing it?
```suggestion
  pkg info --prefix "$package" 2>/dev/null | cut -wf2
```

Reply:
Thanks for noticing this silly code :) Fixed it.